handbook/engineering/hiring/engineering-manager-backend-platform.md

```diff
@@ -0,0 +1,63 @@
+- Regularly communicate the team's progress toward their goals as well as changes in team goals to appropriate stakeholders.
+- Support and coach teammates to grow in their careers and fulfill their responsibilities.
+- Grow the team in a sustainable way so that the team can accomplish more over time.
+
+## Qualifications
+
+- You have 2+ years of experience directly managing at least 4 full-time software engineers. We expect you already know how to lead a team.
+- You have done some amount of coding in the past 2 years (for example: at work or on a personal project) to keep your skills up-to-date. We expect that you are able to review and provide valuable feedback on code, architecture, and designs.
+- You are able to clearly communicate your past accomplishments verbally (example: in an interview) and in writing (examples: in your resume, in your answers to our application questions).
+- You have experience breaking up complex technical problems into smaller achievable milestones that the team can iterate on.
+
+## Learn more about us
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](../../../company/remote/index.md) that is [distributed across the world](../../../company/team/index.md). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](../../people-ops/compensation.md) and [practical benefits](../../people-ops/benefits-and-perks.md) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](../../index.md)
+
+## Interview process
+
+<!-- 1. You [apply here](#todo). --><!-- This is commented out because I will be sending some outbound messages before accepting applications. -->
+
+1. Intro calls
+   - 45m [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) (VP Engineering)
+   - 30m [Tomás Senart](../../../company/team/index.md#tomás-senart) (Engineering Manager, Cloud)
+1. Interviews.
+   - 1h [Engineering leadership](engineering-leadership.md).
+     - The interviewers will be [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) and another Engineering Manager except Tomás.
+   - 1h [Technical experience](https://github.com/sourcegraph/interviews/blob/master/engineering/technical-experience.md) - We ask you about your past work and accomplishments.
+     - The interviewers will be two of the following:
+       - [Ryan Slade](../../../company/team/index.md#ryan-slade-he-him)
+       - [Alex Russell-Saw](../../../company/team/index.md#alex-russell-saw-he-him)
+       - [Thorsten Ball](../../../company/team/index.md#thorsten-ball-he-him)
+   - 1h [Team collaboration](https://github.com/sourcegraph/interviews/blob/master/engineering/team-collaboration.md). We ask you about how you work and communicate in a team setting, and how you handle tricky situations.
```

Comment:
I am considering having one of the interviewers of this module be an EM too (to get another EM in front of the candidate, as well as because for this role there will be cross eng team collaboration too).